runtime/vm/jvminit.c

```diff
@@ -8097,6 +8097,25 @@ setSignalOptions(J9JavaVM *vm, J9PortLibrary *portLibrary)
 		}
 	}
 
+#if defined(WIN32)
+#if defined(_WIN32_WINNT_WINBLUE) && (_WIN32_WINNT_MAXVER >= _WIN32_WINNT_WINBLUE)
+	HMODULE h_kernel32 = GetModuleHandle(TEXT("kernel32.dll"));
+
+	if (h_kernel32 && IsWindows8OrGreater()) {
+		void *getProcessMitigationPolicyFunc = (void*)GetProcAddress(h_kernel32, "GetProcessMitigationPolicy");
```

Comment:
Suggest introducing a typdef, so fewer casts are needed:
```c
		typedef (BOOL (WINAPI * PMP_func)(HANDLE, PROCESS_MITIGATION_POLICY, PVOID, SIZE_T);
		PMP_func getProcessMitigationPolicyFunc = (PMP_func)GetProcAddress(h_kernel32, "GetProcessMitigationPolicy");
```
Please make null checks explicit and split the condition according to our coding standard:
```c
		if (((PMP_func)NULL != getProcessMitigationPolicyFunc)
			&& getProcessMitigationPolicyFunc(
					GetCurrentProcess(),
					ProcessControlFlowGuardPolicy,
					&cfgPolicy,
					sizeof(cfgPolicy))
			&& cfgPolicy.EnableControlFlowGuard
		) {
```

Reply:
Addressed in [86ef934](https://github.com/eclipse-openj9/openj9/compare/e54773912ed222cc3494eb5099f34732506f3f74..86ef93416ee606fe8d7854b5e3f4715a44f6f0bb)